### DISCUSSION TITLE
How to edit short-cut key for tools?

### DISCUSSION THREAD

**Discussion by MRDuangsup** [xournalpp/xournalpp#5609] [Q&A]
I want to edit short-cut key of tools :
default : pen = ctrl+shift+e 
I want to change it , How to do?

If someone already asked, I'm so sorry.

**Comment by saxyx** [comment]
If you have `xournalpp-1.2.3+dev-nightly.20240429-windows` installed, You can change the shortcut in `C:\Program Files\Xournal++\share\xournalpp\ui\mainmenubar.xml`

![image](https://github.com/xournalpp/xournalpp/assets/50071422/17e0d03f-5834-49f7-8252-3da843d52144)
